src/main/java/org/opentripplanner/routing/edgetype/rentedgetype/DropoffVehicleEdge.java

```diff
@@ -0,0 +1,49 @@
+package org.opentripplanner.routing.edgetype.rentedgetype;
+
+import org.opentripplanner.routing.core.State;
+import org.opentripplanner.routing.core.StateEditor;
+import org.opentripplanner.routing.core.vehicle_sharing.VehicleDescription;
+import org.opentripplanner.routing.graph.Vertex;
+
+import java.util.Locale;
+
+/**
+ * This edge is used to allow dropping off rented vehicles at any street vertex we want.
+ */
+public class DropoffVehicleEdge extends EdgeWithParkingZones {
+
+    public DropoffVehicleEdge(Vertex v) {
+        super(v, v);
+    }
+
+    @Override
+    public String getName() {
+        return "Rent vehicle in node " + getToVertex().getName();
```

Comment:
Zmień string na `"Drop off vehicle in node "`

Reply:
done